pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -52,6 +52,16 @@ type VPCNetworkConfigurationSpec struct {
 type VPCNetworkConfigurationStatus struct {
 	// Conditions describes current state of VPCNetworkConfiguration.
 	Conditions []Condition `json:"conditions"`
+	// LBSubnets describes lb subnets info, now it is needed for AKO.
+	LBSubnets []LBSubnetInfo `json:"lbSubnets,omitempty"`
+}
+
+// LBSubnetInfo defines lb subnet info.
+type LBSubnetInfo struct {
+	// NSX PolicyPath for the load balancer Subnet.
+	LBSubnetPath string `json:"lbSubnetPath,omitempty"`
```

Comment:
As this is for AVI only, we can call it `SESubnetPath`? Or how it is called in NSX API?

Reply:
Reminder.

Reply:
Changed to AVISESubnetPath. In NSX, we get it from "Path" field of /orgs/default/projects/<project id>/vpcs/<vpc id>/subnets/<avi se subnet>

Reply:
I meant the name of the Subnet in NSX

Reply:
From NSX: 
          "resource_type": "VpcSubnet",
            "id": "_AVI_SUBNET--LB",
            "display_name": "_AVI_SUBNET--LB",
Shall we use "AVISubnetPath"?

Reply:
Ok, I think either AVISESubnetPath or AVISubnetPath is fine